Shared/Coordinator/MainCoordinator.swift

```diff
@@ -94,24 +94,24 @@ class MainCoordinator: Coordinator<Void> {
         case .home, .conversation:
 #if IOS
             Task {
-                await self.runConversationListFlow()
+                await self.runConversationListFlow(with: deeplink)
```

Comment:
I'm passing the deep link to this function so that it doesn't lose reference to it. Because it's asynchronous, self.deepLink can get set to nil before we present the conversation list.